.travis/build.sh

```diff
@@ -38,19 +38,27 @@ mvn spotbugs:check
 # Run testsuite with java 8 only
 if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
 
-  docker pull oryd/hydra:v1.0.0
+  docker pull oryd/hydra:v1.8.5
+  docker pull quay.io/keycloak/keycloak:latest
```

Comment:
Can we use some specific version instead of `:latest`?

Reply:
Makes sense.